### PR TITLE
Build Test - MOEI Styles, Navbar and Footer

### PR DESCRIPTION
**Pull request by lucasmbispo**

### CODE REVIEW

File: components/home/Nav.tsx [hunk 1]
```diff
@@ -25,79 +25,85 @@ const NavBar: React.FC<any> = ({ settings }) => {
     { title: t('dataset'), path: '/search' },
     { title: t('organization'), path: '/organization', matchExp: '/@' },
     { title: t('topics'), path: '/topic' },
+    { title: 'SDG Data', path: '/topic' },
```
Comment (anuveyatsu) [inline components/home/Nav.tsx:28]: @lucasmbispo is this based on their additional requirements?

File: public/images/uae_moei_en.png
Comment (anuveyatsu) [inline public/images/uae_moei_en.png:1]: @lucasmbispo is this what they provided?

**Review by anuveyatsu — COMMENTED** [review]
Generally LGTM @lucasmbispo well done 👍🏼  I've left couple of comments/questions, please, take a look.

File: components/home/Nav.tsx (hunk 1 above)
Comment (lucasmbispo) [reply components/home/Nav.tsx:28]: Yes @anuveyatsu based on the shared document.

File: public/images/uae_moei_en.png
Comment (lucasmbispo) [reply public/images/uae_moei_en.png:1]: It's also provided in the documents.

https://docs.google.com/document/d/1Qz5TNtBfAJoIghV7_rLr0gwyCy5GYnI6vfZLGQZ2yxM/edit

**Pull request merged**